a_Calculate_Centers/src/calculate_AK_poi.R

```diff
@@ -0,0 +1,129 @@
+#' @title Calcuate POI center for NHD Best Resolution waterbodies from The 
```

Comment:
question: Should the title here mention AK?

Reply:
It absolutely should!